docs/src/components/home/Hero.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import KeyboardArrowRightRounded from '@material-ui/icons/KeyboardArrowRightRounded';
+import ContentCopyRounded from '@material-ui/icons/ContentCopyRounded';
+import GradientText from 'docs/src/components/GradientText';
+
+const Hero = () => {
+  return (
+    <Box sx={{ overflow: 'hidden' }}>
+      <Container sx={{ minHeight: 500, height: 'calc(100vh - 120px)', maxHeight: 700 }}>
+        <Grid
+          container
+          alignItems="center"
+          wrap="nowrap"
+          sx={{ height: '100%', maxWidth: { xs: 500, md: 'initial' }, mx: 'auto' }}
+        >
+          <Grid item md={7} lg={6}>
+            <Box textAlign={{ xs: 'center', md: 'left' }} maxWidth={500}>
+              <Typography
+                variant="h1"
+                fontWeight={800}
+                fontSize="clamp(2.625rem, 1.2857rem + 3.5714vw, 4.5rem)"
```

Comment:
Off topic. It makes me think of https://github.com/mui-org/material-ui/issues/15251

Reply:
I think it is a nice to have in Joy typography, something like
```js
<Typography fontSize="fluid.h1">Big Heading</Typography>
```

Reply:
We could also have the logic bundled into the theme, as we do it in https://next.material-ui.com/customization/typography/#responsive-font-sizes

```suggestion
<Typography variant="h1">Big Heading</Typography>
```

Reply:
So, should we move this style to the theme? How is it going to looking for the other pages? How do we avoid the duplication?

Reply:
Good point. I will put this in another PR when we have another page.